x-pack/plugins/fleet/public/search_provider.ts

```diff
@@ -40,19 +40,45 @@ export const toSearchResult = (
   pkg: PackageListItem,
   application: ApplicationStart,
   basePath: IBasePath
-): GlobalSearchProviderResult => ({
-  id: pkg.name,
-  type: packageType,
-  title: pkg.title,
-  score: 80,
-  icon: getEuiIconType(pkg, basePath),
-  url: {
-    path: `${application.getUrlForApp(INTEGRATIONS_PLUGIN_ID)}${
-      pagePathGetters.integration_details_overview({ pkgkey: pkg.name })[1]
-    }`,
-    prependBasePath: false,
-  },
-});
+): GlobalSearchProviderResult[] => {
+  const packageResult = {
+    id: pkg.name,
+    type: packageType,
+    title: pkg.title,
+    score: 80,
+    icon: getEuiIconType(pkg, basePath),
+    url: {
+      path: `${application.getUrlForApp(INTEGRATIONS_PLUGIN_ID)}${
+        pagePathGetters.integration_details_overview({ pkgkey: pkg.name })[1]
+      }`,
+      prependBasePath: false,
+    },
+  };
+
+  const policyTemplateResults = pkg.policy_templates?.map<GlobalSearchProviderResult>(
+    (policyTemplate) => ({
+      id: policyTemplate.name,
+      type: packageType,
+      title: policyTemplate.title,
+      score: 80,
+      icon: getEuiIconType(pkg, basePath, policyTemplate),
+      url: {
+        path: `${application.getUrlForApp(INTEGRATIONS_PLUGIN_ID)}${
+          pagePathGetters.integration_details_overview({
+            pkgkey: pkg.name,
+            integration: policyTemplate.name,
+          })[1]
+        }`,
+        prependBasePath: false,
+      },
+    })
+  );
+
+  return [
+    packageResult,
+    ...(policyTemplateResults && policyTemplateResults.length > 1 ? policyTemplateResults : []),
```

Comment:
when there's only a single package policy - it is not shown in the integration screens.
Therefore, we are not returning it as search result as well

Reply:
suggestion to add that as a comment in the code, it can look like a mistake on first glance

you can also do
```suggestion
    ...(policyTemplateResults?.length > 1 ? policyTemplateResults : []),
```